README.md

```diff
@@ -344,6 +344,76 @@ If set to true, docker compose will build with the `--with-dependencies` option
 
 The default is `false`.
 
+#### `builder` (object)
+
+Defines the properties required for creating, using and removing Builder Instances. If not set, the default Builder Instance on the Agent Instance will be used.
+
+##### `bootstrap` (boolean)
+
+If set to true, will boot builder instance after creation. Optional when using `create`.
+
+The default is `true`.
+
+##### create (boolean)
+
+If set to true, will use `docker buildx create` to create a new Builder Instance using the propeties defined.
+
+The default is `false`.
+
+##### debug (boolean)
+
+If set to true, enables debug logging during creation of builder instance. Optional when using `create`.
+
+The default is `false`.
+
+##### `driver` (string)
+
+If set will create a Builder Instance using the selected Driver and use it. Available Drivers:
+
+- `docker-container` creates a dedicated BuildKit container using Docker.
+- `kubernetes` creates BuildKit pods in a Kubernetes cluster.
+- `remote` connects directly to a manually managed BuildKit daemon.
+
+More details on different [Build Drivers](https://docs.docker.com/build/builders/drivers/).
+
+##### `driver-opt` (string)
+
+Optional, commas separated, Key-Value pairs of driver-specific options to configure the Builder Instance when using `create`. Available options for each Driver:
+
+- [docker-container](https://docs.docker.com/build/builders/drivers/docker-container/)
+- [kubernetes](https://docs.docker.com/build/builders/drivers/kubernetes/)
+- [remote](https://docs.docker.com/build/builders/drivers/remote/)
+
+Example: `memory=100m`
```

Comment:
I would make the example use more than one option to show how to comma-separate them